scripts/case_selection/export_bpc_selected_cases.R

```diff
@@ -128,18 +128,18 @@ patient_output$redcap_repeat_instance <- rep("")
 
 patient_output$genie_patient_id <- patient_output$record_id
 patient_output$birth_year <- clinical$birth_year[match(patient_output$genie_patient_id, clinical$patient_id)]
-patient_output$naaccr_ethnicity_code <- clinical$ethnicity[match(patient_output$genie_patient_id, clinical$patient_id)]
-patient_output$naaccr_race_code_primary <- clinical$primary_race[match(patient_output$genie_patient_id, clinical$patient_id)]
-patient_output$naaccr_race_code_secondary <- clinical$secondary_race[match(patient_output$genie_patient_id, clinical$patient_id)]
-patient_output$naaccr_race_code_tertiary <- clinical$tertiary_race[match(patient_output$genie_patient_id, clinical$patient_id)]
-patient_output$naaccr_sex_code <- clinical$sex[match(patient_output$genie_patient_id, clinical$patient_id)]
+patient_output$naaccr_ethnicity_code <- clinical$ethnicity_detailed[match(patient_output$genie_patient_id, clinical$patient_id)]
```

Comment:
@danlu1 thoughts on adding some integration tests to make sure we are doing these mappings correctly?

Reply:
I'm thinking about put the mapping lines into a remap_naaccr_code function then test if the function generates expected result. What do you think?

Reply:
I like that idea! Thanks!